src/talk/gst.rs

```diff
@@ -27,8 +27,9 @@ pub(super) fn mic_input(
     volume: f32,
     block_align: u16,
     sample_rate: u16,
+    input_src:&str,
```

Comment:
Also what's your editor? I notice that your not formatting the files quite correctly. There's a missing space after 
`input_src:`

You can fix common issues like this with `cargo fmt`. Some editor like atom have plugins to handle this for you on save.

I also suggest you run code through a linter like `cargo clippy` it will have nice helpful suggestions and I learned quite a bit of rust by reading up on why clippy suggested some changes